JotunnLib/Managers/LocalizationManager.cs

```diff
@@ -228,32 +228,28 @@ private void AddLanguageFilesFromPluginFolder()
         /// <param name="forceReplace">replace the token if it already exists</param>
         public void AddToken(string token, string value, string language = DefaultLanguage, bool forceReplace = false)
         {
-            if (token[0] != TokenFirstChar)
```

Comment:
why did you strip that? for the tokens it should be fine.

Reply:
Wouldn't we want to treat single tokens in the same way that we treat an entire localization dictionary though?
If we can add a dictionary that doesn't require tokens to start with `$`, I would think that adding single tokens should behave the same way

Reply:
OK, if you are sure of that change, lets add it :)